docs/blog/20230114-confused-Techie-v1.101.0-beta.md

```diff
@@ -30,8 +30,8 @@ And as always, happy coding, see you among the stars!
 
 - Fixed a bug where macOS menus like "Open" don't do anything
 - Fixed a bug where macOS wouldn't open files by dragging them onto the dock.
-- Fixed a bug where devtools won't open (https://github.com/pulsar-edit/pulsar/issues/260)
-- Fixed a bug where the editor refused to open with the message "GPU process isn't usable. Goodbye" (https://github.com/pulsar-edit/pulsar/issues/233)
+- Fixed a bug where devtools won't open (<https://github.com/pulsar-edit/pulsar/issues/260>)
```

Comment:
What is the reason for adding `<>` around these links? This text has been copied directly from our actual change log. So it'd be important to know if we have to start manually editing the ChangeLog just to work on the website